index.html

```diff
@@ -38,6 +39,131 @@ <h1 class="primary-text">I’m Richmond Glad to see you!</h1>
+            <p class="card-text">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <div class="button1"><a href="#">See Project</a></div>
+          </div>
+          <div class="card">
+            <div class="img"><img src="./logos/t3.png" alt="work example" /></div>
+            <p class="card-title">Tonic</p>
+            <div class="tag">
+              <ul>
+                <li>Canopy</li>
+                <li>Backend-Dev</li>
+                <li> 2015</li>
+            </ul>
+             </div>
+            <p class="card-text">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <div class="button1"><a href="#">See Project</a></div>
+          </div>
+          <div class="card">
+            <div class="img"><img src="./logos/t4.png" alt="work example" /></div>
+            <p class="card-title">Multi-Post Stories</p>
+            <div class="tag">
+              <ul>
+                <li>Canopy</li>
+                <li>Backend-Dev</li>
+                <li> 2015</li>
+            </ul>
+             </div>
+            <p class="card-text">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <div class="button1"><a href="#">See Project</a></div>
+          </div>
+        </div>
+      </section>
+      <section class="container2">
+        <p class="primary-text">About Me</p>
+        <p class="content">
+            I’m a software developer! I can help you build a product , 
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+        </p>
+        <p class="Connect">LET'S CONNECT</p>
+        <ul class="social-media">
+            <li class="medium"><a href="#"><img src="./logos/medium.png" alt="medium" /></a>
+            </li>
+            <li class="github"><a href="#"><img src="./logos/github.png" alt="github" /></a>
+            </li>
+            <li class="angelistic"><a href="#"><img src="./logos/angelist.png" alt="angelistic" /></a>
+            </li>
+            </ul>
+             <div class="button1"><a href="#">Get My Resume</a></div>
+
+            <div class="skill-title">
+              <div class="container1">
+                <h1 class="skill">Languages</h1>
+                <h1 class="dropdown">--</h1>
```

Comment:
- [ ] You have multiple `h1` tags on your page. It is better for your page to have just one h1 tag, this makes your website semantic and easily readable by browsers. Please use other tags instead of `h1` here.